src/main/java/fr/onagui/alignment/container/SKOSContainer.java

```diff
@@ -554,4 +558,34 @@ public Resource getRoot() {
 		}
 		return this.conceptSchemes;
 	}
-}
\ No newline at end of file
+
+	@Override
+	public Optional<Date> getModifiedDate(Resource cpt) {
+
+		Optional<Date>date=null;
```

Comment:
Use "empty()" instead of null
https://www.quora.com/What-is-the-difference-between-null-and-optional-empty-in-Java8